CHANGELOG.md

```diff
@@ -23,4 +23,6 @@ Changes should start with one of the following tags:
 - [removed] The `issue_report_channel` key was removed
 - [removed] The `cache` key was removed
 - [removed] The `radio_show` key was removed
+- [removed] The `api` key was removed
```

Comment:
Can somebody explain to me why we remove the `api` key? If one want's to stay compatible with 0.13 and 0.14 it has to be allowed to be there in 0.14, just not be required. I'd leave it in and just state that it is just for 0.13 compatibility.

Reply:
Because unknown fields are okay. We're suggesting that people prefix those fields with `ext_` to have not interference with later versions, but from the implementation all additional fields are valid. 

This way we don't have to drag old stuff with us and can let it stay in older versions.

Reply:
I still think it is helpful as a documentation to keep the field, mark it as deprecated, link to the new field `api_compatibility` with an explanation and remove it in 0.15.

Reply:
That would mean that we have to document the `api` field in the 0.14 with no specification if we would require to be 0.14 like it is now it will break with 0.13, if we require it to be 0.13 it will break with 0.12 (not as bad, but putting 0.13 in your 0.14 schema definition feels a lot more confusion potential then just ditching it). 
I would just leave it.

Reply:
I agree that it can be removed to avoid unnecessary noise, but we should document "unknown fields are okay" in the changelog and in the documentation. That *is* a change in how the API is being handled, previously the spec stated that custom fields must start with "ext_".

Reply:
For the website documentation update, I'll provide a pull request.

Reply:
> not as bad, but putting 0.13 in your 0.14 schema definition feels a lot more confusion potential then just ditching it

OK I can see that point. Let's just document it really well and provide an example of a JSON which is compatible with both, 0.13 and 0.14, and a migration guide.